lib/rubocop/cop/mixin/configurable_numbering.rb

```diff
@@ -7,7 +7,7 @@ module Cop
     module ConfigurableNumbering
       include ConfigurableEnforcedStyle
 
-      SNAKE_CASE = /^@{0,2}[-_a-z]+[_\D]*(_\d)*[!?=]?$/
+      SNAKE_CASE = /^@{0,2}[-_a-z]+[_\D]*(_\d*)*[!?=]?$/
```

Comment:
Good catch. This regex needs a bit more tweaking. Changing the `*` to `+?` still matches something ending with `_`. I think this is because of the `[_\D]*` just in front of it.